exercises/nucleotide-codons/description.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-Write a function that returns the name of an amino acid a particular codon, possibly using shorthand, encodes for.
+Write some code which will output the name of an amino acid encoded by a particular codon, possibly using shorthand.
```

Comment:
Perhaps

```suggestion
Produce the name of an amino acid encoded by a particular codon, possibly using shorthand.
```

or, more radically,

```suggestion
Translate a given codon into the name of the amino acid that it encodes, possibly using shorthand.
```

Reply:
I'm going to Bravely Fix it to be like your second suggestion. That reworking had occurred to me too but I was being cautious and trying to use as light a touch as possible. 😅